engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/WorkflowExecutionActor.scala

```diff
@@ -185,6 +195,15 @@ case class WorkflowExecutionActor(params: WorkflowExecutionActorParams)
   when(WorkflowExecutionFailedState) { FSM.NullFunction }
   when(WorkflowExecutionAbortedState) { FSM.NullFunction }
 
+  var previousHeartbeatTime: Option[OffsetDateTime] = None
+  def measureTimeBetweenHeartbeats(): Unit = {
+    val now = OffsetDateTime.now
+    previousHeartbeatTime foreach { previous =>
+      sendGauge(NonEmptyList("workflows", List("workflowexecutionactor", "heartbeat", "interval_millis", "set")), now.toInstant.toEpochMilli - previous.toInstant.toEpochMilli)
```

Comment:
Once this goes into production, would we need to create a new graph in Grafana to show these new metrics or would they appear in any existing graphs?

Reply:
We would have to make new charts for them